service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java

```diff
@@ -156,4 +177,24 @@ public static Applicant expectedApplicant() {
             .build();
     }
 
+    public static Applicant expectedApplicant(final String number) {
```

Comment:
Again duplication here. Couldn't you invoke this method from the oone above passing appropriate parameters?